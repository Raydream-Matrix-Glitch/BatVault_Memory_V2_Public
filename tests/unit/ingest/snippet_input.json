{
  "decisions": {
    "d1": {
      "id": "philips-exit-tv-2011",
      "option": "Exit TV manufacturing",
      "rationale": "Margins in the commodity TV market had eroded below the firm's cost of capital, prompting a strategic exit to redeploy capital toward higher-margin LED lighting and healthcare businesses.",
      "timestamp": "2011-01-15T12:00:00Z"
    }
  },
  "events": {
    "e1": {
      "id": "phil-e6",
      "summary": "Philips posts Q4 loss on TV business",
      "description": "Net loss attributed to price wars in flat-panel TVs.",
      "timestamp": "2010-11-30T09:00:00Z"
    }
  },
  "transitions": {
    "t1": {
      "id": "trans-phil-2011-2013",
      "from": "philips-exit-tv-2011",
      "to": "philips-led-lighting-focus-2013",
      "relation": "causal",
      "reason": "Capital redeployed to LED lighting expansion",
      "timestamp": "2011-12-31T00:00:00Z"
    }
  }
}
