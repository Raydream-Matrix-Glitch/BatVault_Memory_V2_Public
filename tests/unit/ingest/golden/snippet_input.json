[
  {
    "id": "evt-demo-snippet",
    "summary": "Demo event with snippet",
    "description": "A demo event used to satisfy the golden snippet fixture.",
    "timestamp": "2024-01-01T00:00:00Z",
    "snippet": "Demo event with snippet."
  }
]
